### ISSUE TITLE
c:\go\pkg\tool\windows_amd64\cgo.exe: exit status 2

### ISSUE THREAD

**Issue by mathieucaroff** [go-vgo/robotgo#318]
- Robotgo version (or commit ref): 988cd3273de21a5a809e35d4179fe60fdce9da4f (master as of 2021-02-12)
- Go version: `go1.14.3 windows/amd64`
- Gcc version: `gcc.exe (x86_64-posix-seh-rev0, Built by MinGW-W64 project) 8.1.0`
- Operating system and bit: Windows version 2004 / OS Build 19041.804
- Resolution: 1920x1080
- Can you reproduce the bug using the example go code:
  - [ ] Yes (provide example code)
  - [ ] No
  - [v] Not relevant

Environement:

```
> env | grep GO | grep -Eo '^[^=]*'
GOPATH
GOPROXY
INCLUDE
LOGONSERVER
```

## Description

`go get github.com/go-vgo/robotgo` errors with `c:\go\pkg\tool\windows_amd64\cgo.exe: exit status 2`

To be more specific, in powershell the following happens:

```
> go get github.com/go-vgo/robotgo
c:\go\pkg\tool\windows_amd64\cgo.exe: exit status 2
```

```
> cd go/src/github.com/go-vgo/robotgo
> go build
go build github.com/robotn/gohook: c:\go\pkg\tool\windows_amd64\cgo.exe: exit status 2
```

## MinGW was installed using choco

In an elevated powershell:

```
# Install choco
Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
```

Reopen an elevated powershell and:

```
# Install mingw
choco install mingw
# Install go
choco install golang
```

Finally, in a user-level powershell:

```
# Install robotgo
go get github.com/go-vgo/robotgo
```

**Comment by vcaesar** [comment]
You can try with #324.

**Comment by vcaesar** [comment]
Closed because of duplicates, #100. All code tests and C dependencies use MinGw-64, TDM-GCC has not been tested.
But you can compile `lpng and lzib` yourself, and you can download `https://sourceforge.net/projects/mingw-w64/files/ x86_64-posix`.
